test/FastTests/Voron/Trees/FreeSpaceTest.cs

```diff
@@ -246,5 +247,81 @@ public void CanGetListOfAllFreedPages(int maxPageNumber, int numberOfFreedPages,
                 Assert.Equal(sorted, retrievedFreePages);
             }
         }
+
+
+        [Fact]
```

Comment:
This should fail AllTestsShouldUseRavenFactOrRavenTheoryAttributes test?

Reply:
Fixed